### ISSUE TITLE
REST API:  Issue overwriting a parameter "description" in a swagger operation

### ISSUE THREAD

**Issue by bfcamara** [dotnet/docfx#817]
When overwriting a parameter "description", all parameters get the new description

Here is an example

```
...
                "parameters": [
                    {
                        "name": "db",
                        "in": "path",
                        "description": "",
                        "required": true,
                        "type": "integer",
                        "format": "int32"
                    },
                    {
                        "name": "accountKey",
                        "in": "query",
                        "description": "",
                        "required": true,
                        "type": "string"
                    }
                ],~
...
```

And the the overwrite md
```
---
uid: my-uid
summary: a summary
parameters:
    - name: db
      description: The database number
    - name: accountKey
      description: The account key
---
```

And here is the result

![image](https://cloud.githubusercontent.com/assets/822851/19648060/beb156aa-99f8-11e6-9f39-c319b8b69bfc.png)

**Comment by bfcamara** [comment]
@hellosnow Thank you very much.

**Comment by hellosnow** [comment]
The fix will be released at version 2.6.4
